### ISSUE TITLE
Can't stop output to CLI

### ISSUE THREAD

**Issue by JCron245** [yargalot/gulp-accessibility#16]
I was attempting to make this task **only** report to a file and not output to the CLI, but it seems to ignore the verbose option. It properly makes the report file but I was trying to find a way to stop the CLI output. 

```
gulp.task('wcag_report', function() {
  gulp.src(html_paths)
  .pipe(access({
    force: true,
    verbose: false,
  }))
  .pipe(access.report({reportType: 'txt'}))
  .pipe(rename({
    prefix: 'wcag',
    basename: '-report',
    extname: '.txt'
  }))
  .pipe(gulp.dest('reports/'));
});
```

**Comment by yargalot** [comment]
That would be an issue with AccessSniff, Will have a look over there

**Comment by yargalot** [comment]
https://github.com/yargalot/AccessSniff/commit/e5d3e4ed795a5bd09b6eb11ecb3d81eb80108d55

**Comment by yargalot** [comment]
Fixed in 2.4.6
